packages/theme/src/abstracts/_colors.scss

```diff
@@ -3,26 +3,30 @@
 
 // Palette Colors
 // - Grays
```

Comment:
We should remove colors that are duplicated in `_variables-bootstrap.scss` to prevent unclear overrides from taking place.